client.go

```diff
@@ -503,16 +506,16 @@ func (c *Client) monitor(ctx context.Context) {
 						// populated in the previous step.
 
 						activeSubs = 0
-						for _, id := range subsToRepublish {
-							if err := c.republishSubscription(ctx, id, availableSeqs[id]); err != nil {
-								dlog.Printf("republish of subscription %d failed", id)
-								subsToRecreate = append(subsToRecreate, id)
+						for _, subId := range subsToRepublish {
```

Comment:
It should be `subID` but I don't see how this makes it more readable?

Reply:
it doesn't overshadow the id import at top of the file. Again linter error, but I can leave it as is.

Reply:
Ah, then this is a good catch. Please change to `subID` then.

Reply:
This is still `subId`

Reply:
Whoops, sorry, could have sworn I changed it, somehow got lost duing my switches...